CorsixTH/Lua/objects/doors/swing_door_right.lua

```diff
@@ -39,18 +39,14 @@ function SwingDoor:SwingDoor(hospital, object_type, x, y, direction, etc)
   self.is_master = object_type == object
   self:Door(hospital, object_type, x, y, direction, etc)
   if self.is_master then
-    -- Wait one tick before finding the slave so that we're sure it has been created.
-    local --[[persistable:swing_door_creation]] function callback()
-      local slave_type = "swing_door_left"
-      self.slave = self.world:getObject(x - 1, y, slave_type) or self.world:getObject(x, y - 1, slave_type) or nil
-      self.slave:setAsSlave(self)
-      self.ticks = false
-    end
-    self:setTimer(1, callback)
-    self.ticks = true
+    local slave_type = "swing_door_left"
+    self.slave = self.world:getObject(x - 1, y, slave_type) or self.world:getObject(x, y - 1, slave_type) or nil
+    self.slave:setAsSlave(self)
+    assert(self.slave.master, "Swing doors are not paired at " .. x .. "," .. y)
   end
   self.old_anim = self.th:getAnimation()
   self.old_flags = self.th:getFlag()
+  local --[[persistable:swing_door_creation]] function _() end -- Stub from Oct 2023, savegame version 181
```

Comment:
Can you make a wiki note in coding conventions for when we need to use a stub? I appreciate the use of the savegame version as the comment